From ab12096d4f0c858aa61291c97fba2bb138fd55bf Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@nic.cz>
Date: Fri, 26 Jun 2026 15:17:47 +0200
Subject: [PATCH] shadow: add parameter lintl to LIBS

In final link is missing -lintl parameter, this is why we get
compilation error:

getdef.c:(.text+0x654): undefined reference to `libintl_gettext'
.../bin/ld.bfd: ../libsubid/.libs/libsubid.a(libshadow_la-getdef.o):getdef.c:(.text+0x77c): more undefined references to `libintl_gettext' follow
collect2: error: ld returned 1 exit status

Adding it in config variables fixes it.
---
 utils/shadow/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/shadow/Makefile b/utils/shadow/Makefile
index 52845336e..2187f824e 100644
--- a/utils/shadow/Makefile
+++ b/utils/shadow/Makefile
@@ -35,6 +35,7 @@ SHADOW_APPLETS := \
 	useradd userdel usermod vipw
 
 CONFIGURE_ARGS += \
+	LIBS="-lintl" \
 	--without-audit \
 	--without-libpam \
 	--without-selinux \
-- 
2.47.3

